package.json

```diff
@@ -8,9 +8,12 @@
     "dev": "webpack --progress --config webpack.dev.config.babel.js --watch",
     "dev:server": "webpack-dev-server --progress --config webpack.dev.config.babel.js --port 8001 --host 0.0.0.0",
     "build": "npm run clean && npm run build:plugins && npm run build:index",
-    "build:plugins": "webpack --progress --config webpack.prod.config.babel.js && npm run build:index",
+    "build:plugins": "webpack --progress --config webpack.prod.config.babel.js",
```

Comment:
@craftzdog, I wasn't sure why `build:index` was included here, since that made it run twice on `npm run build`. I also included some Windows specific commands so I could run the build on Windows 10.